polybar-scripts/openvpn-status/README.md

```diff
@@ -0,0 +1,32 @@
+# Script: openvpn-status
+
+Show the running openvpn config file
+
+![VPN down](screenshots/down.png)
+
+![VPN connected](screenshots/connected.png)
+
+## Dependencies
```

Comment:
I guess we can delete this. 
Everyone should know that he need OpenVPN ifhe want to use this script.